src/components/Blog/Search/SearchResult/styles.module.css

```diff
@@ -0,0 +1,113 @@
+.searchResult {
+  box-sizing: border-box;
+  display: none;
+  margin-top: 4px;
+  width: 100%;
+  padding: 20px 8px 0 20px;
+  position: absolute;
+  z-index: 5;
+  left: 0;
+  top: 100%;
+  box-shadow: 0 1px 0 0 rgb(0 0 0 / 20%), 0 2px 3px 0 rgb(0 0 0 / 10%);
+  border: 1px solid var(--color-lighter-blue);
+  border-radius: 10px;
+  background: white;
+}
+
+.searchResultShow {
+  display: block;
+}
+
+.searchResultContent {
+  padding-right: 4px;
+  max-height: 650px;
+  overflow: scroll;
```

Comment:
How about `overflow-y: auto`? `-y` so we don't get a horizontal scrollbar and `auto` so it hides when there's only a few results.